packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -175,9 +175,93 @@ export const guides = [
         title: "Sign Users In",
         subLinks: [
           {
-            title: "Add an external Identity Provider",
-            path: "/docs/guides/identity-providers/",
+            title: "Add an Identity Provider",
+            subLinks: [
+              {
+                title: "Overview",
+                path: "/docs/guides/identity-providers/",
+              },
+              {
+                title: "Social Logins",
+                subLinks: [
+                  {
+                    title: "Amazon",
+                    path: "/docs/guides/social-login/amazon/main/"
+                  },
+                  {
+                    title: "Apple",
+                    path: "/docs/guides/add-an-external-idp/apple/main/"
+                  },
+                  {
+                    title: "Facebook",
+                    path: "/docs/guides/add-an-external-idp/facebook/main/"
+                  },
+                  {
+                    title: "GitHub",
+                    path: "/docs/guides/social-login/github/main/"
+                  },
+                  {
+                    title: "GitLab",
+                    path: "/docs/guides/social-login/gitlab/main/"
+                  },
+                  {
+                    title: "Google",
+                    path: "/docs/guides/add-an-external-idp/google/main/"
+                  },
+                  {
+                    title: "LinkedIn",
+                    path: "/docs/guides/add-an-external-idp/linkedin/main/"
+                  },
+                  {
+                    title: "Microsoft",
+                    path: "/docs/guides/add-an-external-idp/microsoft/main/"
+                  },
+                  {
+                    title: "OpenID Connect",
```

Comment:
take this one out and leave the dup in the Enterprise section......

Reply:
Fixed! Thanks @susanharper-okta. With this removed, are you OK to give this one your blessing now?